lib/elixir/pages/anti-patterns/design-anti-patterns.md

```diff
@@ -32,9 +32,9 @@ end
 iex> AlternativeInteger.parse("13")
 13
 iex> AlternativeInteger.parse("13", discard_rest: true)
-13
-iex> AlternativeInteger.parse("13", discard_rest: false)
 {13, ""}
+iex> AlternativeInteger.parse("13", discard_rest: false)
+13
```

Comment:
I think we need to fix the code instead? We need to swap the if and else branches. WDYT?

Reply:
yeah, thanks, you're right. fixed 🤗